std/special/bootstrap.zig

```diff
@@ -101,9 +101,11 @@ inline fn callMainWithArgs(argc: usize, argv: [*][*]u8, envp: [][*]u8) u8 {
 }
 
 extern fn main(c_argc: i32, c_argv: [*][*]u8, c_envp: [*]?[*]u8) i32 {
-    var env_count: usize = 0;
-    while (c_envp[env_count] != null) : (env_count += 1) {}
-    const envp = @ptrCast([*][*]u8, c_envp)[0..env_count];
+    const envp = if (builtin.os != builtin.Os.nspire) blk: {
+        var env_count: usize = 0;
+        while (c_envp[env_count] != null) : (env_count += 1) {}
+        break :blk @ptrCast([*][*]u8, c_envp)[0..env_count];
+    } else [][*]u8{};
```

Comment:
`envp` as a third argument to main is [non-standard](https://stackoverflow.com/a/10321482/9236675), so I added this to fix the error I was getting when I started the program and aborted because `c_envp` wasn't provided to `main`.

Reply:
`envp` should just be completely removed, as Zig provides other ways to get that information. But that can happen after this patch.